File: Dermatoloska_ordinacija/UI/dermatoloska_mobile/lib/models/novost.dart
import 'package:json_annotation/json_annotation.dart';

part 'novost.g.dart';

@JsonSerializable()
class Novost{
	int? novostId;
  String? naslov; 
  String? sadrzaj; 
  DateTime? datumObjave; 
  int? korisnikId;

  Novost(this.novostId, this.naslov, this.sadrzaj, this.datumObjave, this.korisnikId);

  factory Novost.fromJson(Map<String, dynamic> json) => _$NovostFromJson(json);

  Map<String, dynamic> toJson() => _$NovostToJson(this);

}

File: Dermatoloska_ordinacija/DermatoloskaOrdinacija.Services/Database/Novost.cs
using System;
using System.Collections.Generic;

namespace DermatoloskaOrdinacija.Services.Database;

public partial class Novost
{
    public int NovostId { get; set; }

    public string? Naslov { get; set; }

    public string? Sadrzaj { get; set; }

    public DateTime? DatumObjave { get; set; }

    public int? KorisnikId { get; set; }

    public virtual Korisnik? Korisnik { get; set; }
}
